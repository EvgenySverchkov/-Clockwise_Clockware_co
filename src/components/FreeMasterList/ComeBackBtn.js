import React from "react";
import { Link } from "react-router-dom";
import PropTypes from "prop-types";

const ComeBackBtn = ({ backTo }) => (
  <div className="form-group float-left mt-3">
    <Link to={backTo} className="btn btn-primary">
      Ð¡ome back
    </Link>
  </div>
);

ComeBackBtn.propTypes = {
  backTo: PropTypes.string.isRequired,
};

export default ComeBackBtn;
