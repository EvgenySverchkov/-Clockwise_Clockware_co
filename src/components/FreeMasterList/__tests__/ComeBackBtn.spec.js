import React from "react";
import {create} from "react-test-renderer";

import {MemoryRouter, Route} from "react-router-dom";
import ComeBackBtn from "../ComeBackBtn";

it("Test of <ComeBackBtn/> for <FreeMasterList/>", ()=>{
    const mockProps = {
        backTo: "/"
    }
    let component = create(
        <MemoryRouter initialEntries={['/']}>
            <Route path="/">
                <ComeBackBtn {...mockProps}/>
            </Route>
        </MemoryRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
});