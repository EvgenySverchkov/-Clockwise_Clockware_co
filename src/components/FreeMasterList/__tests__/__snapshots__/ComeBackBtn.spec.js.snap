// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <ComeBackBtn/> for <FreeMasterList/> 1`] = `
<div
  className="form-group float-left mt-3"
>
  <a
    className="btn btn-primary"
    href="/"
    onClick={[Function]}
  >
    Ð¡ome back
  </a>
</div>
`;
